draft-ietf-quic-multipath.md

```diff
@@ -232,41 +246,88 @@ the use of the multipath extension during the connection handshake,
 as specified in {{QUIC-TRANSPORT}}. The new transport parameter is
 defined as follows:
 
-- enable_multipath (current version uses 0x0f739bbc1b666d06): the
-  enable_multipath transport parameter is included if the endpoint supports
-  the multipath extension as defined in this document. This parameter has
-  a zero-length value.
-
-If any of the endpoints does not advertise the enable_multipath transport
+- initial_max_paths (current version uses 0x0f739bbc1b666d07): the
```

Comment:
Probably better: `initial_max_path_id`. This would mirror the meaning of the frame as suggested in https://github.com/quicwg/multipath/pull/292/files#r1527500161.

Reply:
My 2c: There's only so much value you get from consistency with RFC 9000. This has created a fair amount of confusion in the review of this PR, and it seems like calling it `initial_max_path_id` (and the corresponding renaming of the frame), would make it 100% clear what is meant, without the need for long explanations.

Reply:
Well, then I think that the frame type should be `MAX_PATH_ID` to further reduce the confusion and increase the multipath specification consistency.

Reply:
Record in issue #324